rules/windows/execution_initial_access_foxmail_exploit.toml

```diff
@@ -0,0 +1,66 @@
+[metadata]
+creation_date = "2024/08/29"
+integration = ["endpoint", "windows"]
+maturity = "production"
+updated_date = "2024/08/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the Foxmail client spawning a child process with argument pointing to the Foxmail temp directory. 
+This may indicate the successful exploitation of a Foxmail vulnerability for initial access and execution via 
```

Comment:
Any CVE specifically?

Reply:
looked into fomail CVE database not accurate (last one goes back to 2008). the blog contains the vuln details.